### PR TITLE
Michaeljeffrey/ch9907/downlink queued item should no longer appear

### PR DESCRIPTION
**Pull request by michaeldjeffrey**

### CODE REVIEW

File: src/device/router_device_worker.erl [hunk 1]
```diff
@@ -885,6 +885,7 @@ handle_info(
                 DownlinkPacket,
                 Region
             ),
+            ok = maybe_send_queue_update(Device1, State),
```
Comment (macpie) [inline src/device/router_device_worker.erl:888]: Idk about this one as we do not actually update the queue but rather potentially remove a downlink from it

Comment (michaeldjeffrey) [reply src/device/router_device_worker.erl:888]: When we send a downlink they don't want to see it in the side panel any more.

Comment (michaeldjeffrey) [reply src/device/router_device_worker.erl:888]: Unless it's a confirmed down. But once it's out of the queue, they want that shown on the frontend

Comment (macpie) [reply src/device/router_device_worker.erl:888]: sold

**Review by macpie — APPROVED** [review]

**Pull request merged**